assets/js/components/settings/SettingsCardConsentMode.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Site Kit by Google, Copyright 2023 Google LLC
```

Comment:
```suggestion
 * Site Kit by Google, Copyright 2024 Google LLC
```

😅

Reply:
Those copy/pastas inevitably sneak in :sweat_smile: